site/public/ts/shared/controllers/HeaderController.ts

```diff
@@ -128,6 +128,14 @@ export class HeaderController {
 
             main.setLocked(!main.isLocked());
         });
+
+        $(".cookie__banner__accept").click(() => {
+            const banner = document.getElementById("cookie-banner");
+            banner.classList.add("invisible");
```

Comment:
Should just do `$("#cookie-banner").addClass("invisible")` since already using JQuery